sdk/communication/communication-call-automation/test/node/callMediaClient.spec.ts

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 // External module imports
```

Comment:
nit: remove Line 3 and probably Line 6 too as comments no longer make sense.

Reply:
Fxied